website/docs/guides/1.0.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.0.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-100-upgrade-guide"
+description: |-
+    MongoDB Atlas Provider 0.1.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.0.0: Upgrade and Information Guide
+
+## Deprecation of Project IP Whitelist and import with Project IP Access List
```

Comment:
We already deprecated it so this is the EoL, perhaps better said as:
 ## End of Life and removal of Project IP Whitelist and import with Project IP Access List